pages/k8s/upgrading.md

```diff
@@ -19,9 +19,9 @@ toc: False
   <p markdown="1" class="p-notification__response">
     <span class="p-notification__status">Note:</span>
 This page describes the general upgrade process. It is important to follow the specific upgrade pages for each release, as these may include additional steps and workarounds for safely upgrading. <br><br>
-<a class='p-button--brand' href='/kubernetes/docs/1.19/upgrading'>Upgrade to 1.19 </a>
-<a class='p-button--brand' href='/kubernetes/docs/1.18/upgrading'>Upgrade to 1.18 </a>
-<a class='p-button--brand' href='/kubernetes/docs/1.17/upgrading'>Upgrade to 1.17 </a>
+<a class='p-button--brand' href='/kubernetes/docs/1.19/upgrading'>Upgrade to 1.20 </a>
```

Comment:
looks like the hrefs are pointing to `n-1` for this and the next 2 links.

Reply:
thx

Reply:
@kwmonroe fixed